### ISSUE TITLE
NullpointerException when using unrecognized Renderer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [0xfab-ri/universal-media-server#29]
```
What steps will reproduce the problem?
1. Try to play a video using a renderer that can not be recognized. 


What is the expected output? What do you see instead?
That the video plays.

What version of the product are you using? On what operating system?
2.2.1 on Windows Server 2012 64 bit

Please provide any additional information below.
Log say that the renderer could not be recognized. 
NullpointerException in thread StartPlayingEvent at DLNAResource.java:1552
another NullpointerExcption at in thread StopPlayingEvent at 
DLNAResource.java:1614

```

Original issue reported on code.google.com by `michael....@gmail.com` on 29 Dec 2012 at 12:34

**Comment by GoogleCodeExporter** [comment]
```
Does this still happen in 2.2.5?
```

Original comment by `subjunk` on 19 Jan 2013 at 10:11

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I can't test it, because I don't have this renderer anymore. 
```

Original comment by `michael....@gmail.com` on 22 Jan 2013 at 12:27

**Comment by GoogleCodeExporter** [comment]
```
This issue will be fixed in the next version after 2.2.6
```

Original comment by `subjunk` on 22 Jan 2013 at 10:59
- Changed state: **Fixed**
